src/lib/fabric.ts

```diff
@@ -0,0 +1,18 @@
+import { fabric } from "fabric";
```

Comment:
i feel like morally we should be PRing to DefinitelyTyped/DefinitelyTyped

Reply:
We should but any changes made there should be definitively correct and I certainly don't know that yet